hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/processor/InsertWatermarkP.java

```diff
@@ -29,7 +29,7 @@
 
 /**
  * A processor that inserts watermark into a data stream. See
```

Comment:
"insert watermark_s_". Btw there are quite a few instances of "insert watermark" in samples.